data/lib/core/player.lua

```diff
@@ -737,3 +742,37 @@ function Player.disableLoginMusic(self)
 	msg:delete()
 	return true
 end
+
+function Player.isCharmUnlocked(self, charmId)
+	if self:getStorageValue(PlayerStorageKeys.charmsUnlocked + charmId) == 1 then
```

Comment:
Storages now return nil if empty, doing a direct check with a number is wrong

Reply:
that is why this PR is on draft, i porting it from my personal project to TFS